vaadin-renderer-flow-parent/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.vaadin</groupId>
+        <artifactId>vaadin-flow-components</artifactId>
+        <version>21.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>vaadin-renderer-flow-parent</artifactId>
+    <packaging>pom</packaging>
+    <name>Flow Renderer Parent</name>
+    <modules>
+        <module>vaadin-renderer-flow</module>
+        <module>vaadin-renderer-flow-integration-tests</module>
```

Comment:
```
<profile>
            <id>default</id>
            <activation>
                <property>
                    <name>!release</name>
                </property>
            </activation>
            <modules>
                <module>vaadin-renderer-flow-integration-tests</module>
            </modules>
        </profile>
  ```
this is not correct.  this module should be removed from here, as under there is a profile to add it to the maven reactor. 

i think that is the reason why the build failed for this PR.

Reply:
Thanks, that did it. Now the tests passed

Reply:
great